src/ace.F90

```diff
@@ -228,13 +228,14 @@ subroutine read_ace_table(i_table, i_listing)
     integer, intent(in) :: i_table   ! index in nuclides/sab_tables
     integer, intent(in) :: i_listing ! index in xs_listings
 
+    integer       :: in = UNIT_DATAFILE ! file unit
```

Comment:
Rather than create a new hard-coded unit, we should just be using `open(NEWUNIT=...)`.